### PR TITLE
feat: update peer

### PR DESCRIPTION
**Pull request by nachomazzara**

### CODE REVIEW

File: .ci/.env.prd [hunk 1]
```diff
@@ -4,7 +4,7 @@ PUBLIC_URL=/
 REACT_APP_CHAIN_ID=1
 REACT_APP_LOCAL_STORAGE_KEY=builder
 REACT_APP_EMAIL_SERVER_URL=https://decentraland.org/subscribe
-REACT_APP_PEER_URL=https://peer.decentraland.org
+REACT_APP_PEER_URL=https://peer-lb.decentraland.org
```
Comment (nachomazzara) [inline .ci/.env.prd:7]: [`getCollectionBaseURI`](https://github.com/decentraland/builder/blob/9dddc44ef63a372d84ec55520702f35bd06f893b/src/modules/collection/utils.ts#L55) needs to use `peer.decentraland.org`

File: .ci/.env.stg [hunk 2]
```diff
@@ -4,7 +4,7 @@ PUBLIC_URL=/
 REACT_APP_CHAIN_ID=1
 REACT_APP_LOCAL_STORAGE_KEY=builder
 REACT_APP_EMAIL_SERVER_URL=https://decentraland.org/subscribe
-REACT_APP_PEER_URL=https://peer.decentraland.org
+REACT_APP_PEER_URL=https://peer-lb.decentraland.org
```
Comment (nachomazzara) [inline .ci/.env.stg:7]: [`getCollectionBaseURI`](https://github.com/decentraland/builder/blob/9dddc44ef63a372d84ec55520702f35bd06f893b/src/modules/collection/utils.ts#L55) needs to use `peer.decentraland.org`

File: .ci/.env.prd (hunk 1 above)
Comment (nicosantangelo) [reply .ci/.env.prd:7]: you mean that the method should prefix without the new `-lb` dns?

Comment (nachomazzara) [reply .ci/.env.prd:7]: because that info will go on-chain and will be impossible to be changed

**Pull request merged**